### PR TITLE
Blog app - integration specs for views

### PR DESCRIPTION
**Pull request by sheylaPozo**
# Integration specs for views
### Gems:

- Install "database_cleaner" gem
- Configure spec_helper.rb and rails_helper.rb for proper using of default installed in Rails v 6, "capybara" gem.
- Interaction Tests using "capybara":

### Login session:

- Test the view of the browser depending on different cases of filled-in data.

### User:

**Testing if everything renders properly on the pages:**

- user#index.
- user#show

### Testing if everything renders properly on the pages:

- post#index.
- post#show

> Thank you for your nice review 😄❄

### CODE REVIEW

File: spec/features/login/login_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'Login Page', type: :feature do
```
Comment (sinansevgi) [inline spec/features/login/login_spec.rb]: - [x] You should test whether the application redirects to the root page when the correct username and password are entered. 

Example
```ruby
expect(current_path).to eq(root_path)
```

File: spec/features/user/user_index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
```
Comment (sinansevgi) [inline spec/features/user/user_index_spec.rb]: - [x] You should have a test case that asserts you can see the profile picture for each user.

File: spec/features/user/user_show_spen.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
```
Comment (sinansevgi) [inline spec/features/user/user_show_spen.rb]: - [x] You should have a test case that asserts you can see the user's profile picture.
- [x] You should have a test case that asserts you can see the user's first 3 posts.
- [x] You should have a test case that checks if the app redirects to the post's show page when you click on a user's post.

File: spec/features/post/post_index_spec.rb [hunk 4]
```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
```
Comment (sinansevgi) [inline spec/features/post/post_index_spec.rb]: - [x] You should have a test case that asserts you can see the user's profile picture.

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @sheylaPozo, 

Great Work!
Most of the requested test scenarios ok :clap: . 
There are some issues that you still need to work on to go further but you are almost there!



 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/features/login/login_spec.rb (hunk 1 above)
Comment (sheylaPozo) [reply spec/features/login/login_spec.rb]: > Done ✔

File: spec/features/user/user_index_spec.rb (hunk 2 above)
Comment (sheylaPozo) [reply spec/features/user/user_index_spec.rb]: Done ✔

File: spec/features/user/user_show_spen.rb (hunk 3 above)
Comment (sheylaPozo) [reply spec/features/user/user_show_spen.rb]: Done ✔

File: spec/features/post/post_index_spec.rb (hunk 4 above)
Comment (sheylaPozo) [reply spec/features/post/post_index_spec.rb]: Done ✔

**Comment by Davidosky007** [comment]
## Hi @sheylaPozo 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
job-welldone-gif




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sheylaPozo** [comment]
Thank youu :) @Davidosky007

**Pull request merged**